templates/prometheus.go

```diff
@@ -45,25 +52,50 @@ var PrometheusTemplate = promv1.Prometheus{
 				Port:      intstr.FromString("web"),
 			}},
 		},
-		Resources: utils.GetResourceRequirements("prometheus"),
-		// prometheus tls -- will be uncommented
-		// Web: &promv1.WebSpec{
-		// 	TLSConfig: &promv1.WebTLSConfig{
-		// 		KeySecret: corev1.SecretKeySelector{
-		// 			Key: "tls.key",
-		// 		},
-		// 		Cert: promv1.SecretOrConfigMap{
-		// 			Secret: &corev1.SecretKeySelector{
-		// 				Key: "tls.crt",
-		// 			},
-		// 		},
-		// 		ClientAuthType: fmt.Sprint(tls.VerifyClientCertIfGiven),
-		// 		ClientCA: promv1.SecretOrConfigMap{
-		// 			ConfigMap: &corev1.ConfigMapKeySelector{
-		// 				Key: "service-ca.crt",
-		// 			},
-		// 		},
-		// 	},
-		// },
+		Resources:   utils.GetResourceRequirements("prometheus"),
+		ListenLocal: true,
+		Containers: []corev1.Container{{
+			Name:  "kube-rbac-proxy",
+			Image: "quay.io/brancz/kube-rbac-proxy:v0.12.0",
+			Args: []string{
+				"--secure-listen-address=0.0.0.0:9339",
+				"--upstream=http://127.0.0.1:9090/",
+				"--logtostderr=true",
+				"--v=10",
```

Comment:
- too much verbosity i guess 😅

Reply:
I used the one used by in-cluster monitoring, I don't have any problem in reducing it but it helps to debug.